common/lib/xmodule/xmodule/js/src/video/02_html5_video.js

```diff
@@ -16,11 +17,82 @@
 (function(requirejs, require, define) {
     define(
 'video/02_html5_video.js',
-[],
-function() {
+['underscore'],
+function(_) {
```

Comment:
have you tried not to import _ ? i've seen _ being used without defining them in each files.